### ISSUE TITLE
Crashes before it even starts 

### ISSUE THREAD

**Issue by bandit67890** [overviewer/Minecraft-Overviewer#1339]
installed overviewer from APT repository (minecraft-overviewer_0.12.137-0~overviewer1_amd64.deb) anytime I attempt to run it, I get the following error, even without any arguments.  just running overview.py gives this:  (same with arguments, even just -h)

> /usr/bin/overviewer.py -h
> Traceback (most recent call last):
>   File "/usr/bin/overviewer.py", line 38, in <module>
>     from overviewer_core import logger
> ImportError: cannot import name logger
>

**Comment by eminence** [comment]
Can you show us the output of `ls -l /usr/lib/python2.{6,7}/dist-packages/overviewer_core/`  ?

**Comment by bandit67890** [comment]
```
/usr/lib/python2.6/dist-packages/overviewer_core/:
total 120
lrwxrwxrwx 1 root root    58 Sep 27 01:53 assetmanager.py -> ../../../../share/pyshared/overviewer_core/assetmanager.py
drwxr-xr-x 2 root root  4096 Nov  7 20:10 aux_files
lrwxrwxrwx 1 root root    51 Sep 27 01:53 cache.py -> ../../../../share/pyshared/overviewer_core/cache.py
lrwxrwxrwx 1 root root    58 Sep 27 01:53 configParser.py -> ../../../../share/pyshared/overviewer_core/configParser.py
-rw-r--r-- 1 root root 94568 Sep 27 01:53 c_overviewer.so
drwxr-xr-x 5 root root  4096 Nov  7 20:10 data
lrwxrwxrwx 1 root root    56 Sep 27 01:53 dispatcher.py -> ../../../../share/pyshared/overviewer_core/dispatcher.py
lrwxrwxrwx 1 root root    51 Sep 27 01:53 files.py -> ../../../../share/pyshared/overviewer_core/files.py
lrwxrwxrwx 1 root root    54 Sep 27 01:53 __init__.py -> ../../../../share/pyshared/overviewer_core/__init__.py
lrwxrwxrwx 1 root root    51 Sep 27 01:53 items.py -> ../../../../share/pyshared/overviewer_core/items.py
lrwxrwxrwx 1 root root    52 Sep 27 01:53 logger.py -> ../../../../share/pyshared/overviewer_core/logger.py
lrwxrwxrwx 1 root root    49 Sep 27 01:53 nbt.py -> ../../../../share/pyshared/overviewer_core/nbt.py
lrwxrwxrwx 1 root root    54 Sep 27 01:53 observer.py -> ../../../../share/pyshared/overviewer_core/observer.py
lrwxrwxrwx 1 root root    60 Sep 27 01:53 optimizeimages.py -> ../../../../share/pyshared/overviewer_core/optimizeimages.py
lrwxrwxrwx 1 root root    64 Sep 27 01:53 overviewer_version.py -> ../../../../share/pyshared/overviewer_core/overviewer_version.py
lrwxrwxrwx 1 root root    57 Sep 27 01:53 progressbar.py -> ../../../../share/pyshared/overviewer_core/progressbar.py
lrwxrwxrwx 1 root root    50 Sep 27 01:53 rcon.py -> ../../../../share/pyshared/overviewer_core/rcon.py
lrwxrwxrwx 1 root root    57 Sep 27 01:53 rendermodes.py -> ../../../../share/pyshared/overviewer_core/rendermodes.py
lrwxrwxrwx 1 root root    64 Sep 27 01:53 settingsDefinition.py -> ../../../../share/pyshared/overviewer_core/settingsDefinition.py
lrwxrwxrwx 1 root root    64 Sep 27 01:53 settingsValidators.py -> ../../../../share/pyshared/overviewer_core/settingsValidators.py
lrwxrwxrwx 1 root root    53 Sep 27 01:53 signals.py -> ../../../../share/pyshared/overviewer_core/signals.py
lrwxrwxrwx 1 root root    54 Sep 27 01:53 textures.py -> ../../../../share/pyshared/overviewer_core/textures.py
lrwxrwxrwx 1 root root    53 Sep 27 01:53 tileset.py -> ../../../../share/pyshared/overviewer_core/tileset.py
lrwxrwxrwx 1 root root    50 Sep 27 01:53 util.py -> ../../../../share/pyshared/overviewer_core/util.py
lrwxrwxrwx 1 root root    51 Sep 27 01:53 world.py -> ../../../../share/pyshared/overviewer_core/world.py

/usr/lib/python2.7/dist-packages/overviewer_core/:
total 552
lrwxrwxrwx 1 root root     58 Sep 27 01:53 assetmanager.py -> ../../../../share/pyshared/overviewer_core/assetmanager.py
-rw-r--r-- 1 root root   7469 Nov  7 20:10 assetmanager.pyc
drwxr-xr-x 2 root root   4096 Nov  7 20:10 aux_files
lrwxrwxrwx 1 root root     51 Sep 27 01:53 cache.py -> ../../../../share/pyshared/overviewer_core/cache.py
-rw-r--r-- 1 root root   5962 Nov  7 20:10 cache.pyc
lrwxrwxrwx 1 root root     58 Sep 27 01:53 configParser.py -> ../../../../share/pyshared/overviewer_core/configParser.py
-rw-r--r-- 1 root root   5135 Nov  7 20:10 configParser.pyc
-rw-r--r-- 1 root root  98376 Sep 27 01:53 c_overviewer.so
drwxr-xr-x 5 root root   4096 Nov  7 20:10 data
lrwxrwxrwx 1 root root     56 Sep 27 01:53 dispatcher.py -> ../../../../share/pyshared/overviewer_core/dispatcher.py
-rw-r--r-- 1 root root  13953 Nov  7 20:10 dispatcher.pyc
lrwxrwxrwx 1 root root     51 Sep 27 01:53 files.py -> ../../../../share/pyshared/overviewer_core/files.py
-rw-r--r-- 1 root root   5385 Nov  7 20:10 files.pyc
lrwxrwxrwx 1 root root     54 Sep 27 01:53 __init__.py -> ../../../../share/pyshared/overviewer_core/__init__.py
-rw-r--r-- 1 root root   2297 Nov  7 20:10 __init__.pyc
lrwxrwxrwx 1 root root     51 Sep 27 01:53 items.py -> ../../../../share/pyshared/overviewer_core/items.py
-rw-r--r-- 1 root root  10564 Nov  7 20:10 items.pyc
lrwxrwxrwx 1 root root     52 Sep 27 01:53 logger.py -> ../../../../share/pyshared/overviewer_core/logger.py
-rw-r--r-- 1 root root   8541 Nov  7 20:10 logger.pyc
lrwxrwxrwx 1 root root     49 Sep 27 01:53 nbt.py -> ../../../../share/pyshared/overviewer_core/nbt.py
-rw-r--r-- 1 root root  11787 Nov  7 20:10 nbt.pyc
lrwxrwxrwx 1 root root     54 Sep 27 01:53 observer.py -> ../../../../share/pyshared/overviewer_core/observer.py
-rw-r--r-- 1 root root  19967 Nov  7 20:10 observer.pyc
lrwxrwxrwx 1 root root     60 Sep 27 01:53 optimizeimages.py -> ../../../../share/pyshared/overviewer_core/optimizeimages.py
-rw-r--r-- 1 root root   9365 Nov  7 20:10 optimizeimages.pyc
lrwxrwxrwx 1 root root     64 Sep 27 01:53 overviewer_version.py -> ../../../../share/pyshared/overviewer_core/overviewer_version.py
-rw-r--r-- 1 root root    403 Nov  7 20:10 overviewer_version.pyc
lrwxrwxrwx 1 root root     57 Sep 27 01:53 progressbar.py -> ../../../../share/pyshared/overviewer_core/progressbar.py
-rw-r--r-- 1 root root  17990 Nov  7 20:10 progressbar.pyc
lrwxrwxrwx 1 root root     50 Sep 27 01:53 rcon.py -> ../../../../share/pyshared/overviewer_core/rcon.py
-rw-r--r-- 1 root root   3025 Nov  7 20:10 rcon.pyc
lrwxrwxrwx 1 root root     57 Sep 27 01:53 rendermodes.py -> ../../../../share/pyshared/overviewer_core/rendermodes.py
-rw-r--r-- 1 root root  11219 Nov  7 20:10 rendermodes.pyc
lrwxrwxrwx 1 root root     64 Sep 27 01:53 settingsDefinition.py -> ../../../../share/pyshared/overviewer_core/settingsDefinition.py
-rw-r--r-- 1 root root   2971 Nov  7 20:10 settingsDefinition.pyc
lrwxrwxrwx 1 root root     64 Sep 27 01:53 settingsValidators.py -> ../../../../share/pyshared/overviewer_core/settingsValidators.py
-rw-r--r-- 1 root root  17482 Nov  7 20:10 settingsValidators.pyc
lrwxrwxrwx 1 root root     53 Sep 27 01:53 signals.py -> ../../../../share/pyshared/overviewer_core/signals.py
-rw-r--r-- 1 root root   4155 Nov  7 20:10 signals.pyc
lrwxrwxrwx 1 root root     54 Sep 27 01:53 textures.py -> ../../../../share/pyshared/overviewer_core/textures.py
-rw-r--r-- 1 root root 142166 Nov  7 20:10 textures.pyc
lrwxrwxrwx 1 root root     53 Sep 27 01:53 tileset.py -> ../../../../share/pyshared/overviewer_core/tileset.py
-rw-r--r-- 1 root root  49229 Nov  7 20:10 tileset.pyc
lrwxrwxrwx 1 root root     50 Sep 27 01:53 util.py -> ../../../../share/pyshared/overviewer_core/util.py
-rw-r--r-- 1 root root  14572 Nov  7 20:10 util.pyc
lrwxrwxrwx 1 root root     51 Sep 27 01:53 world.py -> ../../../../share/pyshared/overviewer_core/world.py
-rw-r--r-- 1 root root  28350 Nov  7 20:10 world.pyc
```

**Comment by CounterPillow** [comment]
is `/usr/bin/overviewer.py` even from the deb? I thought be used `/usr/bin/overviewer` in those packages but it might just be a symlink. At any rate, are you sure you never installed this locally before by building from source or something?

**Comment by bandit67890** [comment]
I believe it is, because when i do a sudo apt-get purge minecraft-overviewer then /usr/bin/overviewer.py gets deleted.

Yes, I should have mentioned I had a version I had run back in 2011 (last time I was playing minecraft) I think at that time I built it from source, but I have removed the old version of overview.py which was the first problem I ran into.  Perhaps there are more files somewhere that I need to finish cleaning out?  I can't seem to find them.
